src/main/webapp/resources/js/pages/AdminUsersPage.jsx

```diff
@@ -147,7 +147,7 @@ function UsersTable() {
  * @returns {*}
```

Comment:
Can you move this inside the admin directory, want to keep everything specifically tied to the admin in it's own grouping.